recipes/openmmtorchplugin/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+pytorch_version:
+  - 1.11
```

Comment:
pytorch is already pinned to 1.11 for all packages on conda-forge. Use that variable instead.

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml

Reply:
I have removed this as requested